Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -14,11 +14,13 @@ public enum Route: DerivePartialIsos {
   case admin(Admin)
   case appleDeveloperMerchantIdDomainAssociation
   case blog(Blog)
+  case discounts(organization: String)
   case episode(Either<String, Int>)
   case episodes
   case expressUnsubscribe(userId: Database.User.Id, newsletter: Database.EmailSetting.Newsletter)
   case expressUnsubscribeReply(MailgunForwardPayload)
   case feed(Feed)
+  case fika
```

Comment:
this is just `/fika`. unfortunately we have to support this cause I already printed cards, oops!